common/autoware_motion_utils/include/autoware/motion_utils/trajectory/path_shift.hpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef AUTOWARE__MOTION_UTILS__TRAJECTORY__PATH_SHIFT_HPP_
+#define AUTOWARE__MOTION_UTILS__TRAJECTORY__PATH_SHIFT_HPP_
+
+#include <cmath>
```

Comment:
(minor) this include seems unnecessary.
```suggestion
```
Maybe it should be included in the cpp file instead.

Reply:
Deleted in eefab5f1c01db3bb95ae7dd52ff4d732e92cd856, and rebased this PR. Could you approve the [preceding PR](https://github.com/autowarefoundation/autoware.universe/pull/9081) too?